Makefile

```
@@ -59,6 +59,9 @@ test_ci:
 	echo "Detected $(NPROCS) CPUs running with $(CONCURRENCY) workers"
 	stestr run --concurrency $(CONCURRENCY)
 
+test_randomized:
+	python3 -m unittest discover -s test/randomized -v
```

The job failed because of a 10min without any output, we can just use `travis_wait` in front of this which will make the call async and avoid the timeout with no output.

```suggestion
	travis_wait python3 -m unittest discover -s test/randomized -v
```

---

Oh, actually that's in the wrong place ignore that suggestion. It needs to be in the travis yaml not the makefile.